integration_tests/ci/run-all.sh

```diff
@@ -106,10 +111,10 @@ stop_rest_server() {
 set -euo pipefail
 
 log "Compiling local binaries"
-make install &>${DETAILED_OUTPUT_TARGET}
+#make install &>${DETAILED_OUTPUT_TARGET}
```

Comment:
Is it commented intentionally?

Reply:
@ashcherbakov - Commented by mistake, I was re-running the tests on local so commented for speed.